### DISCUSSION TITLE
How to reverse ordering of bibliography?

### DISCUSSION THREAD

**Discussion by petermacp** [typst/typst#3293] [Show and tell]
I am writing my CV using Typst within a .qmd file in RStudio. I need to have my academic publications listed in reverse chronological order (i.e. most recent first), but can't seem to achieve this. Grateful for any suggestions

```{=typst}

#v(10pt, weak: true)

// Summary
#align(left, text(12pt, rgb("#01497c"))[
  *Publications*
])

#line(length: 100%, stroke: 0.5pt)

#align(left, text(10pt, rgb("#212529"), weight: "light")[

#bibliography("data/my_pubs.bib", full: true, style: "american-medical-association-no-et-al.csl", title: "")

])

#v(25pt, weak: true)

```

**Comment by frozolotl** [comment]
I have modified the CSL file accordingly: https://gist.github.com/frozolotl/218e1c156b0c78a3c1003d4c0dda846e

At the bottom of that page is a diff showing what I changed to sort the bibliography entries.
If you want to learn more about sorting in CSL, check out the docs[^1].

[^1]: https://docs.citationstyles.org/en/stable/specification.html#sorting

**Comment by petermacp** [comment]
That is brilliant - thanks so much, and for showing me how to edit the CSL file. Very much appreciated!

**Comment by petermacp** [comment]
Thanks so much - really appreciated, and also for showing me how to edit the .CSL file!
